src/internal/packager/helm/repo.go

```diff
@@ -174,8 +182,14 @@ func (h *Helm) DownloadPublishedChart(cosignKeyPath string) error {
 		},
 	}
 
-	// Download the file (we don't control what name helm creates here)
-	saved, _, err := chartDownloader.DownloadTo(chartURL, pull.Version, h.chartPath)
+	// Download the file into a temp directory since we don't control what name helm creates here
+	temp := filepath.Join(h.chartPath, "temp")
+	if err = utils.CreateDirectory(temp, 0700); err != nil {
+		return fmt.Errorf("unable to download the helm chart: %w", err)
```

Comment:
this is an improper error message for this operation, `unable to create temp directory: %w` would be a bit better

Reply:
done